comprehensive/lms/templates/navigation.html

```diff
@@ -220,7 +220,9 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
 <%include file="help_modal.html"/>
 ## Don't show cookie-banner if already accepted.
 <script type="text/javascript">
+
     if (cookieNotice.getCookie("cookie-banner") == "true") {
```

Comment:
also, do we need to inline js in the html file? i'd rather move this to the js file if possible. in fact, we don't have to register the on click events if this conditional is true. in other words, if (cookie) { closeBanner(); } else { /*register on click events*/ }

the jquery off will should be a no-opt so it's safe to execute them everytime closeCookieBanner() is called (as we have it now)